.travis.yml

```diff
@@ -1,12 +1,15 @@
 language: go
 go:
   - 1.7
```

Comment:
Currently, it's actually not a big deal for M3DB - 1.8 isn't terrible.  Some other services have issues it seems but not M3DB.

Regardless we should definitely be at least testing with 1.8.